lint-gitlab-ci.sh

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env bash
 set -e
 
-GITLAB_SERVER_URL=${GITLAB_SERVER_URL:-"gitlab.com"}
+# GITLAB_SERVER_URL=${GITLAB_SERVER_URL:-"gitlab.com"}
+# ## get gitlab push remote
+GITLAB_REMOTE=$(git remote -v | grep push | awk '{print $2}')
```

Comment:
Regarding your comment (https://github.com/FalcoSuessgott/lint-gitlab-ci/pull/10#discussion_r1324368214) 

> ah I see. I tested it with a repository that I recently moved to another group, but did not update the git remotes. I guess that is something we could document on the README.
> 
> However, nice job!

Note also how `GITLAB_REMOTE` is retrieved, by taking the push part of `git remote -v`.
I haven't checked how it looks if there are more remotes configured. I guess it would be good to pipe it through `head -n1` or something like that.